src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/CreateNewListModal.tsx

```diff
@@ -220,7 +254,7 @@ export const CreateNewListModal: FC<CreateNewListModalProps> = ({
   )
 }
 
-export const CreateNewListModalContainer: FC<CreateNewListModalContainerProps> = props => {
+export const CreateNewListModalContainer: React.FC<CreateNewListModalContainerProps> = props => {
```

Comment:
**question**: @dimatretyak I remember that you prefer using `FC` instead of `React.FC` and you asked me to change `React.FC` to `FC` in one of my PRs recently. I don't have strong opinion what to use, but let's all agree on something

Reply:
@nickskalkin good catch. @gkartalis could you please use just `FC<CreateNewListModalContainerProps>`?

Reply:
🤷‍♂️ sure thing!

Reply:
thanks!